tests/wpt/runner/expectation.json

```diff
@@ -10993,5 +10993,209 @@
+        "new DOMMatrixReadOnly(\"unset\")",
+        "new DOMMatrixReadOnly(\"scale(2, 2), translateX(5px) translateY(5px)\")"
+      ],
+      "DOMMatrix-002.html": true,
+      "DOMMatrix-003.html": true,
+      "DOMMatrix-a-f-alias.html": true,
+      "DOMMatrix-attributes.html": true,
+      "DOMMatrix-css-string.worker.html": true,
+      "DOMMatrix-invert-invertible.html": true,
+      "DOMMatrix-invert-non-invertible.html": true,
+      "DOMMatrix-invert-preserves-2d.html": true,
+      "DOMMatrix-newobject.html": true,
+      "DOMMatrix-stringifier.html": [
+        "WebKitCSSMatrix stringifier: identity (2d)",
+        "WebKitCSSMatrix stringifier: identity (3d)",
+        "WebKitCSSMatrix stringifier: NaN (2d)",
+        "WebKitCSSMatrix stringifier: NaN (3d)",
+        "WebKitCSSMatrix stringifier: Infinity (2d)",
+        "WebKitCSSMatrix stringifier: Infinity (3d)",
+        "WebKitCSSMatrix stringifier: -Infinity (2d)",
+        "WebKitCSSMatrix stringifier: -Infinity (3d)",
+        "WebKitCSSMatrix stringifier: 1/3 (2d)",
+        "WebKitCSSMatrix stringifier: 1/3 (3d)",
+        "WebKitCSSMatrix stringifier: 1/300000 (2d)",
+        "WebKitCSSMatrix stringifier: 1/300000 (3d)",
+        "WebKitCSSMatrix stringifier: 1/300000000 (2d)",
+        "WebKitCSSMatrix stringifier: 1/300000000 (3d)",
+        "WebKitCSSMatrix stringifier: 100000 + (1/3) (2d)",
+        "WebKitCSSMatrix stringifier: 100000 + (1/3) (3d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 1 (2d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 1 (3d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 2 (2d)",
+        "WebKitCSSMatrix stringifier: Math.pow(2, 53) + 2 (3d)",
+        "WebKitCSSMatrix stringifier: Number.MAX_VALUE (2d)",
+        "WebKitCSSMatrix stringifier: Number.MAX_VALUE (3d)",
+        "WebKitCSSMatrix stringifier: Number.MIN_VALUE (2d)",
+        "WebKitCSSMatrix stringifier: Number.MIN_VALUE (3d)",
+        "WebKitCSSMatrix stringifier: throwing getters (2d)",
+        "WebKitCSSMatrix stringifier: throwing getters (3d)"
+      ],
+      "DOMMatrix2DInit-validate-fixup.html": false,
+      "DOMMatrixInit-validate-fixup.html": true,
+      "DOMPoint-001.html": true,
+      "DOMPoint-002.html": true,
+      "DOMQuad-001.html": true,
+      "DOMQuad-002.html": true,
+      "DOMQuad-nan.html": true,
+      "DOMRect-001.html": true,
+      "DOMRect-002.html": true,
+      "DOMRect-nan.html": true,
+      "DOMRectList.html": false,
+      "WebKitCSSMatrix.html": false,
+      "WebKitCSSMatrix.worker.html": true,
+      "historical.html": true,
+      "idlharness.any.html": [
+        "DOMPointReadOnly interface: existence and properties of interface object",
+        "DOMPoint interface: existence and properties of interface object",
+        "DOMRectReadOnly interface: existence and properties of interface object",
+        "DOMRect interface: existence and properties of interface object",
```

Comment:
These tests fail because Web IDL requires [the interface objects to be defined as data properties](https://webidl.spec.whatwg.org/#define-the-global-property-references) but here they are implemented as getters to support lazy loading.